content/influxdb/clustered/admin/licensing.md

```diff
@@ -0,0 +1,170 @@
+    parent: Administer InfluxDB Clustered
+    name: Manage your license
+weight: 101
+influxdb/clustered/tags: [licensing]
+related:
+  - /influxdb/clustered/install/licensing/
+  - /influxdb/clustered/admin/upgrade/
+---
+
+Install and manage your InfluxDB Clustered license to authorize the use of
+the InfluxDB Clustered software.
+
+- [Install your InfluxDB license](#install-your-influxdb-license)
+- [Recover from an unlicensed release](#recover-from-an-unlicensed-release)
+- [Renew your license](#renew-your-license)
+- [License enforcement](#license-enforcement)
+  - [A valid license is required](#a-valid-license-is-required)
+  - [Periodic license checks](#periodic-license-checks)
+  - [License grace periods](#license-grace-periods)
+    - [License expiry logs](#license-expiry-logs)
+    - [Query brownout](#query-brownout)
+
+{{% note %}}
+#### License enforcement is currently an opt-in feature
+
+In currently available versions of InfluxDB Clustered, license enforcement is an
+opt-in feature. This allows InfluxData to introduce license enforcement to a
+limited number of customers in a way that can easily be disabled if issues arise.
+In the future, all releases of InfluxDB Clustered will require an active license
+to use the product.
+
+To opt into license enforcement, enable the `useLicensedBinaries` feature flag
+in your `AppInstance` resource _([See the example below](#enable-feature-flag))_.
+
+{{% /note %}}
+
+## Install your InfluxDB license
+
+{{% note %}}
+If setting up an InfluxDB Clustered deployment for the first time, first
+[set up the prerequisites](/influxdb/clustered/install/licensing/) and
+[configure your cluster](/influxdb/clustered/install/configure-cluster/).
+You will be able to install your license after the `influxdb` namespace is
+created and prepared.
+{{% /note %}}
+
+1.  If you haven't already,
+    [request an InfluxDB Clustered license](https://influxdata.com/contact-sales).
+2.  InfluxData provides you with a `license.yml` file that encapsulates your
+    license token as a custom Kubernetes resource.
+3.  Use `kubectl` to apply and create the `License` resource in your InfluxDB
+    namespace:
+
+    ```sh
+    kubectl apply --filename license.yml --namespace influxdb
+    ```
+
+4.  <span id="enable-feature-flag"></span>
+    Update your `AppInstance` resource to enable the `useLicensedBinaries` feature flag.
+    Add the `useLicensedBinaries` entry to the `.spec.package.spec.featureFlags`
+    property--for example:
+
+    ```yml
+    apiVersion: kubecfg.dev/v1alpha1
+    kind: AppInstance
+    # ...
+    spec:
+      package:
+        spec:
+          featureFlags:
+            - useLicensedBinaries
+    ```
+
+InfluxDB Clustered detects the `License` resource and extracts the credentials
+into a secret required by InfluxDB Clustered Kubernetes pods.
+Pods validate the license secret both at startup and periodically (roughly once
+per hour) while running.
+
+## Recover from an unlicensed release
```

Comment:
```suggestion
## Recover from a license misconfiguration
```

It's a small thing, but the recovery would be from a bad configuration, not from a licensed or unlicensed release. I included it to hopefully allay any fears that license enforcement could lead to service disruptions.